test/routes/routesApi.spec.js

```diff
@@ -811,6 +811,27 @@ describe('Le serveur MSS des routes /api/*', () => {
         testeur.depotDonnees().homologation = () => Promise.resolve(homologation);
       });
 
+      describe('avec un email en majuscules', () => {
+        it('est retrouvé donc son compte ne sera pas recréé', (done) => {
```

Comment:
Nitpick – Le `it` ici veut dire « le serveur MSS », et la sortie texte du test ne veut pas dire grand'chose :
```
test_1  |   Le serveur MSS des routes /api/*
test_1  |     quand requête POST sur `/api/autorisation`
test_1  |       si le contributeur existe déjà
test_1  |         avec un email en majuscules
test_1  |           ✔ est retrouvé donc son compte ne sera pas recréé (87ms)
```

Il serait préférable d'avoir comme nom de test « retrouve le compte et le ne recrée donc pas ».

(Je me charge de corriger.)

Reply:
:+1: